docs/data/charts/zoom-and-pan/zoom-and-pan.md

```diff
@@ -8,7 +8,7 @@ components: ScatterChartPro, BarChartPro, LineChartPro, ChartZoomSlider
 
 <p class="description">Enables zooming and panning on specific charts or axis.</p>
 
-Zooming is possible on the **Pro**[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan') versions of the charts: `<LineChartPro />`, `<BarChartPro />`, `<ScatterChartPro />`.
+Zooming is possible on the Pro version of the charts: `<LineChartPro />`, `<BarChartPro />`, `<ScatterChartPro />`.
```

Comment:
Should we add the nbsp to all chart names? 

Otherwise it can also break in mobile:

![image](https://github.com/user-attachments/assets/b0717532-14ed-4a77-8ef8-d2edf25da303)

Reply:
```suggestion
Zooming is possible on the Pro version of the charts: `<LineChartPro />`, `<BarChartPro />`, `<ScatterChartPro />`.
```

Reply:
Agree, let's do it.